docs/manual/determinism-checker.tex

```diff
@@ -476,7 +488,34 @@
 ondList.remove(0);
 \end{Verbatim}
 Since the list is \<@OrderNonDet>, removing an element at index 0 could remove any of its 3 elements in different runs
-making the resulting list \<@NonDet> which is unsound.
+
+\subsectionAndLabel{Special overriding rule}{determinism-poly-overriding}
+Consider the following code:
+\begin{Verbatim}
+    class Super {
+        void m(@PolyDet String a, @PolyDet("use") String b) {...}
+    }
+    class Sub extends Super {
+        @Override
+        void m(@PolyDet("use") String a, @PolyDet String b) {...}
+    }
+\end{Verbatim}
+Since \code{@PolyDet} is considered to be
+the same as \code{@polyDet("use)} in the determinism type hierarchy, this overriding
+relationship is valid. However, it allows the following unsound instantiations:
+\begin{Verbatim}
+    void caller(@NonDet String nStr, @Det String dStr) {
+        Super superObj = new Super();
+        Sub subObj = new Sub();
+        superObj.m(nStr, dStr);
+        subObj.m(dStr, nStr);
+    }
+\end{Verbatim}
+The first argument of \code{subObj.m} is not a supertype of the
```

Comment:
This does not explain the unsoundness. Why does it matter that the first arg of subObj.m is not a supertype of the first argument of superObjt.m? 

Also, it's confusing that both methods have `@PolyDet("use")`.  Only the method in the sub class should have `@PolyDet("use")`